xnemogcm/nemo.py

```diff
@@ -103,12 +103,15 @@ def nemo_preprocess(ds, domcfg, point_type=None):
     # rename time and space
     # get time_counter bounds
     time_b = ds["time_counter"].attrs.get("bounds")
-    if time_b:
+    if time_b and time_b in ds:
         to_rename.update({"time_counter": "t", time_b: "t_bounds"})
     else:
         to_rename.update({"time_counter": "t"})
+        if not time_b in ds:
```

Comment:
```suggestion
        if time_b not in ds:
```

cf the action that fails
https://github.com/rcaneill/xnemogcm/actions/runs/10808075904/job/29980236483?pr=122#step:4:114